doc/frameworks/tensorflow/using_tf.rst

```diff
@@ -197,6 +197,27 @@ The following args are not permitted when using Script Mode:
 Where the S3 url is a path to your training data within Amazon S3.
 The constructor keyword arguments define how SageMaker runs your training script.
 
+Specify a Docker image using an Estimator
+-----------------------------------------
+
+You can also specify a Docker image when creating an Estimator by specifying the ECR URI for the Python and framework version directly. For a full list of available container URIs, see `Available Deep Learning Containers Images <https://github.com/aws/deep-learning-containers/blob/master/available_images.md>`__ For more information on using Docker containers, see `Use Your Own Algorithms or Models with Amazon SageMaker <https://docs.aws.amazon.com/sagemaker/latest/dg/your-algorithms.html>`__.
+
+When specifying the image, you must use the ``image_name=''`` arg to replace both of the following args:
+
+- ``py_version=''``
+- ``framework_version=''``
+
+The following example uses the ``image_name='763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:1.15.2-cpu-py37-ubuntu18.04'`` arg to specify the container image, Python version, and framework version:
```

Comment:
I think it would be better to put an obviously fake placeholder image URI and direct to the doc page with the updated URIs. While the example URI is the latest TF 1.x version right now, it's less likely that this will get updated as future versions of TF are released.

Reply:
Fixed